lib/rexml/parsers/xpathparser.rb

```diff
@@ -52,11 +52,13 @@ def abbreviate( path )
           when :child
             string << "/" if string.size > 0
           when :descendant_or_self
-            string << "//"
```

Comment:
Why should we revert this?

Reply:
Prior to 399e83d83ab5a9d2a4438fb3379b750261ffb0ec the following script
```ruby
require 'rexml'

def test_abbreviate xpath
  parser = REXML::Parsers::XPathParser.new
  parsed = parser.parse(xpath)
  puts "parsed=#{parsed}"
  test = parser.abbreviate parsed
  puts "expected(#{xpath}) actual(#{test})"
  puts
end

test_abbreviate('.//a/b')
```
outputs:
```
parsed=[:self, :node, :descendant_or_self, :node, :child, :qname, "", "a", :child, :qname, "", "b"]
expected(.//a/b) actual(////a/b)
```
Reverting 399e83d83ab5a9d2a4438fb3379b750261ffb0ec gives the correct output:
```
parsed=[:self, :node, :descendant_or_self, :node, :child, :qname, "", "a", :child, :qname, "", "b"]
expected(.//a/b) actual(.//a/b)
```

Reply:
Hmm, we should have auto tests for `XPathParser#abbrebiate`.